README.md

```diff
@@ -551,4 +551,4 @@ Make sure to consult the [Karabiner.json Reference Manual](https://pqrs.org/osx/
   label: volume_down (equal tovolume_decrement)
 
 - name: volume_up
-  label: volume_up (equal tovolume_increment`)
\ No newline at end of file
+  label: volume_up (equal tovolume_increment`)
```

Comment:
I modified the file in github web ui directly, probably this is just whitespace change